### PR TITLE
Updated to merge into master

### PR DESCRIPTION
**Pull request by vainaviv**

### CODE REVIEW

File: Brain/brain_utils.h [hunk 1]
```diff
@@ -39,18 +40,22 @@ struct valveInfo {
...
 
 valveInfo valve_ids[7] = {
-  {"LOX 2 Way", 20, &(Solenoids::armLOX), &(Solenoids::disarmLOX)} //example
+  {"LOX 2 Way", 20, &(Solenoids::armLOX), &(Solenoids::disarmLOX)}, //example
+  {"LOX 5 Way", 21, &(Solenoids::openLOX), &(Solenoids::closeLOX)},
+  {"LOX GEMS", 22, &(Solenoids::ventLOXGems), &(Solenoids::closeLOXGems)},
+  {"Propane 2 Way", 23, &(Solenoids::armPropane), &(Solenoids::disarmPropane)},
+  {"Propane 5 Way", 24, &(Solenoids::openPropane), &(Solenoids::closePropane)},
+  {"Propane GEMS", 25, &(Solenoids::ventPropaneGems), &(Solenoids::closePropaneGems)},
+  {"High Pressure Solenoid", 26, &(Solenoids::activateHighPressureSolenoid), &(Solenoids::deactivateHighPressureSolenoid)}
```
Comment (fazerlicourice7) [inline Brain/brain_utils.h]: I think this is a mistake. don't want these in master. Just want the first one

File: Brain/brain_utils.h [hunk 2]
```diff
@@ -25,10 +25,11 @@ union floatArrToBytes {
 struct sensorInfo {
   String sensor_name;
   int board_address;
-  int sensor_id;
+  byte sensor_id;
```
Comment (fazerlicourice7) [inline Brain/brain_utils.h]: we just decided to make this an int.

File: Brain/Brain.ino [hunk 3]
```diff
@@ -15,40 +15,37 @@
...
-  {"Low Pressure",8,0,1,1,NULL} //example
+sensorInfo all_ids[11] = {
+  // local sensors
+   {"LOX Injector Low Pressure",  -1, -1, 1, 1, &(Ducers::readLOXInjectorPressure)} //example
 };
 
-sensorInfo sensor = {"",0,0,0,0, NULL};
-
-int numSensors = 1;
+sensorInfo sensor = {"", 0, 0, 0, 0, NULL};
```
Comment (fazerlicourice7) [inline Brain/Brain.ino:31]: keep this variable in. We'll need it.

**Review by fazerlicourice7 — CHANGES_REQUESTED** [review]

File: Brain/brain_utils.h (hunk 1 above)
Comment (vainaviv) [reply Brain/brain_utils.h]: Oops thought I removed these, will update

File: Brain/Brain.ino (hunk 3 above)
Comment (vainaviv) [reply Brain/Brain.ino:31]: For what? I didn't see it in use so I didn't leave it in

File: Brain/brain_utils.h (hunk 2 above)
Comment (vainaviv) [reply Brain/brain_utils.h]: Right right ty

File: Brain/Brain.ino (hunk 3 above)
Comment (fazerlicourice7) [reply Brain/Brain.ino:31]: hmm... I guess you're right. I thought I was using it somewhere

**Review by fazerlicourice7 — APPROVED** [review]

**Pull request merged**